app/models/ship_reviewer_payout_request.rb

```diff
@@ -40,7 +40,7 @@ class ShipReviewerPayoutRequest < ApplicationRecord
   scope :pending_requests, -> { where(status: :pending) }
 
   def self.calculate_amount_for_decisions(decisions_count)
-    (decisions_count / 2) * 0.5
+    decisions_count * ShipReviewerMultiplierService::BASE_SHELLS_PER_REVIEW
```

Comment:
This only pays the base payout-- did you mean for that to happen? It seems like you'd want the multiplier here